cluster/images/etcd-version-monitor/README.md

```diff
@@ -0,0 +1,30 @@
+# etcd-version-monitor
+
+This is a tool for exporting metrics exposed by etcd endpoints that are not
+in prometheus metrics format. The exported metrics would be in prometheus
```

Comment:
This sentence is no longer true - we are exposing some that are already in prometheus format.

Also, this description is in general not accurate. The purpose of this tool is to expose etcd metrics that might be interesting from monitoring purposes in a prometheus format (no matter how do we get them).